sonaranalyzer-dotnet/src/SonarAnalyzer.Common/Rules/Hotspots/DoNotHardcodeCredentialsBase.cs

```diff
@@ -53,8 +53,10 @@ public string CredentialWords
                 this.splitCredentialWords = value.ToUpperInvariant()
                     .Split(new[] { ',' }, StringSplitOptions.RemoveEmptyEntries)
                     .Select(x => x.Trim())
+                    .Select(Regex.Escape)
```

Comment:
Can we add configuration test to prove that we'll not fail with bad config value?